components/calendar/tests/exhaustive.rs

```diff
@@ -47,7 +47,7 @@ fn check_from_fields() {
                 .into_iter()
             })
```

Comment:
Suggestion: Add tests that out-of-range years return RangeError (as opposed to panicking or something)